src/encoded/schemas/library_preparation.json

```diff
@@ -125,6 +125,21 @@
                 ]
             }
         },
+        "restriction_enzymes": {
```

Comment:
I suspect string values are fine here.  in 4DN we have enzyme items as it was deemed important for tracking enzyme source and I believe info about the recognition sites are used - perhaps indirectly - in the processing pipeline for Hi-C.  Just something to consider and maybe ask bioinformatics team about in case they might use info about the particular enzyme used in processing?

Reply:
That's a good point. I will ask!